examples/config/config

```diff
@@ -258,14 +260,20 @@ set history_disable_easter_egg 1
 @cbind  w            = event REQ_NEW_WINDOW
 
 # Page movement binds
-@cbind  j            = scroll vertical 20
-@cbind  k            = scroll vertical -20
-@cbind  h            = scroll horizontal -20
-@cbind  l            = scroll horizontal 20
+@cbind  j            = scroll vertical 100
+@cbind  J            = scroll vertical 50
+@cbind  k            = scroll vertical -100
+@cbind  K            = scroll vertical -50
+@cbind  h            = scroll horizontal -100
+@cbind  H            = scroll horizontal -50
+@cbind  l            = scroll horizontal 100
+@cbind  L            = scroll horizontal 50
 @cbind  <Page_Up>    = scroll vertical -100%
 @cbind  <Page_Down>  = scroll vertical 100%
 @cbind  <Ctrl>f      = scroll vertical 100%
 @cbind  <Ctrl>b      = scroll vertical -100%
+@cbind  gg           = scroll vertical begin
```

Comment:
I like this one and it's the only default binding I replace in my config right now